packages/beacon-node/src/chain/chain.ts

```diff
@@ -505,6 +506,36 @@ export class BeaconChain implements IBeaconChain {
     }
   }
 
+  prunePastInvalidSszObjects(): void {
+    const invalidObjectsDir = this.persistInvalidSszObjectsDir;
+    const invalidObjectsRetentionDays = this.opts.persistInvalidSszObjectsRetention ?? 7;
```

Comment:
I feel like specifying the default value here is not ideal, might be better to define default options object which can also be referenced in the `defaultDescription` in cmd options. That way it is ensured that those are always in sync

https://github.com/ChainSafe/lodestar/blob/751969c03ae443032e9e4d062d1e15ff89a2d00a/packages/cli/src/cmds/beacon/options.ts#L91